streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceivedBlockTracker.scala

```diff
@@ -247,7 +247,8 @@ private[streaming] class ReceivedBlockTracker(
         true
       } catch {
         case NonFatal(e) =>
-          logWarning(s"Exception thrown while writing record: $record to the WriteAheadLog.", e)
+          logWarning(log"Exception thrown while writing record: " +
+            log"${MDC(RECEIVED_BLOCK_TRACKER_LOG_EVENT, record)} to the WriteAheadLog.", e)
```

Comment:
I'm not sure if it's called `RECORD`, maybe it's more `general`.

Reply:
Using the class name `RECEIVED_BLOCK_TRACKER_LOG_EVENT` is also fine